<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointRegView</class>
 <widget class="QWidget" name="PointRegView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_FixedPointsLabel">
         <property name="text">
          <string>Fixed points</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QmitkDataStorageComboBox" name="m_FixedPointsCombo">
         <property name="toolTip">
          <string>select the fixed point set to register to</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_MovingPointsLabel">
         <property name="text">
          <string>Moving points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QmitkDataStorageComboBox" name="m_MovingPointsCombo">
         <property name="toolTip">
          <string>select the point set to be transformed</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="m_PointBasedRegistrationButton">
         <property name="toolTip">
          <string>perform the registration</string>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_RMSError">
         <property name="toolTip">
          <string>rms error</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="ctkMatrixWidget" name="m_MatrixWidget" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="m_ComposeWithDataButton">
         <property name="toolTip">
          <string>composes the transformation with the selected data set</string>
         </property>
         <property name="text">
          <string>Compose with data</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QmitkDataStorageCheckableComboBox" name="m_ComposeWithDataNode">
         <property name="toolTip">
          <string>select a data set</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="m_SaveToFileButton">
         <property name="toolTip">
          <string>saves the current transformation to file</string>
         </property>
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkDataStorageCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageCheckableComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
